pkg/context/vmware/cluster_context.go

```diff
@@ -50,5 +52,39 @@ func (c *ClusterContext) Patch(ctx context.Context) error {
 			vmwarev1.LoadBalancerReadyCondition,
 		),
 	)
-	return c.PatchHelper.Patch(ctx, c.VSphereCluster)
+
```

Comment:
Can we use this function here? because this function is also used in two other reconcilers which reconcile on the vspherecluster object

Reply:
To be honest. I would get rid of this patch func here and just move to the regular patchHelper in all 3 controllers